fs/config/configflags/configflags.go

```diff
@@ -44,6 +44,7 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.BoolVarP(flagSet, &fs.Config.SizeOnly, "size-only", "", fs.Config.SizeOnly, "Skip based on size only, not mod-time or checksum")
 	flags.BoolVarP(flagSet, &fs.Config.IgnoreTimes, "ignore-times", "I", fs.Config.IgnoreTimes, "Don't skip files that match size and time - transfer all files")
 	flags.BoolVarP(flagSet, &fs.Config.IgnoreExisting, "ignore-existing", "", fs.Config.IgnoreExisting, "Skip all files that exist on destination")
+	flags.BoolVarP(flagSet, &fs.Config.B2DisableCheckSum, "b2-disable-checksum", "", fs.Config.B2DisableCheckSum, "Disable b2 checksum")
```

Comment:
This config should go here which makes backend config rather than global config.

https://github.com/ncw/rclone/blob/a2341cc412a0cf4b84c0ec729231e88bc32f5acf/backend/b2/b2.go#L123

You'll then need to add it into the options struct

https://github.com/ncw/rclone/blob/a2341cc412a0cf4b84c0ec729231e88bc32f5acf/backend/b2/b2.go#L136